ui/values/strings.go

```diff
@@ -288,7 +288,7 @@ const (
 	StrDestinationMissing              = "destinationMissing"
 	StrDestinationModalInfo            = "destinationModalInfo"
 	StrDestinationWalletNotSynced      = "destinationWalletNotSynced"
-	StrDex                             = "dex"
+	StrDEX                             = "dex"
```

Comment:
Was this not used before? Don't see previous reference being updated.

Reply:
> Was this not used before?

Yh, looks like it.